CHANGELOG.md

```diff
@@ -34,44 +33,38 @@ Versioning](http://semver.org/spec/v2.0.0.html).
   Omitting `goLiveDate` means there is no date before which the message is not
   live. Omitting `expireDate` means there is no date after which the message is
   expired. (#846)
-+ Use sentence-case rather within `time-sensitive-content` widget type (#832)
++ Use sentence-case within `time-sensitive-content` widget type (#832)
 + Improved spacing/appearance of push-content side navigation for desktop (#836)
 + Mobile search "Close" button now uses arrow icon, so not to be confused with
   "clear" functionality (#836)
 + `action-items` widgets now handle more kinds of errors more fluently (#839)
 
 ### Removed
 
-
-
 ### Deprecated
 
-
-
 ### Documentation
 
-
-
 ## [10.3.0][] - 2018-09-19
 
-### Added
+### Added in 10.3.0
```

Comment:
`in <version>` is extra typing for each release.
Most changelogs I've left it as `### Added` and use the main release header if I need to deep link to a specific release.
But that's just me :man_shrugging:

Reply:
Duplicate headings trigger the linter, but more importantly they break deep link generation in a not-clearly-broken way. Duplicate headings mean I can grab a hyperlink from GitHub.com that won't do what I expect. (Alas, GitHub does not suppress generating the hyperlinks in the ambiguous case.) If I can make that mistake, I probably will, and indeed have.

Reply:
"in <version>" also *might* make the headings churn enough that naive pull request merging won't put changes in the wrong spot in the CHANGELOG so often. Here's hoping.